src/styles/blocks/nav.scss

```
@@ -0,0 +1,46 @@
+.nav {
+  &__list {
+    margin: 0;
+    padding: 0;
+    list-style: none;
+    display: flex;
+    text-transform: uppercase;
+    font-size: 12px;
+  }
+
+  &__list :first-child {
+    margin: 0;
+  }
+  
+  &__item {
+    display: block;
+    margin-left: 20px;
+  }
+  
+  &__link {
+    display: block;
+    height: 60px;
+    line-height: 60px;
+    text-decoration: none;
+    color: $main-accent;
+  }
+  
+  &__link:hover {
+    transition: color 300ms;
+    color: $blue-accent;
+  }
+}
+
+.is-active {
+  color: $blue-accent;
+
+  &:after {
+    position: relative;
+    bottom: 4px;
+    content: "";
+    display: block;
+    height: 4px;
```

should be the full width
```suggestion
    height: 4px;
    width: 100%;
```